lib/l10n/arb/app_pa.arb

```diff
@@ -0,0 +1,81 @@
+{
+  "language":"Panjabi",
+  "@language":{
+    "description":"ਮੌਜੂਦਾ ਭਾਸ਼ਾ"
+  },
+
+
+  "pageNotFound":"ਪੰਨਾ ਮਿਲਿਆ ਨਹੀਂ",
+  "settings":"ਵਿਵਸਥਾ",
+  "enterNewEmail":"ਨਵੀਂ ਈਮੇਲ ਦਾਖਲ ਕਰੋ",
+  "enterRegisteredEmail":"ਰਜਿਸਟਰਡ ਈਮੇਲ ਦਰਜ ਕਰੋ",
+  "enterCurrentPassword":"ਮੌਜੂਦਾ ਪਾਸਵਰਡ ਦਿਓ",
+  "resetPassword":"ਪਾਸਵਰਡ ਰੀਸੈੱਟ",
+  "autoSync":"ਆਟੋ ਸਿੰਕ",
+  "alreadyHaveAnAccount":"ਕੀ ਪਹਿਲਾਂ ਤੋਂ ਹੀ ਖਾਤਾ ਹੈ?",
+  "syncNow":"ਸਿੰਕ",
```

Comment:
This appears to be wrong, its translating to `sink` instead. I got this on google translate `ਹੁਣੇ ਸਿੰਕ ਕਰੋ` can you check once

Reply:
yes, updated now